mne/io/base.py

```diff
@@ -1500,6 +1500,31 @@ def plot_psd_topo(self, tmin=0., tmax=None, fmin=0, fmax=100, proj=False,
                                  axis_facecolor=axis_facecolor, dB=dB,
                                  show=show, n_jobs=n_jobs, verbose=verbose)
 
+    def plot_sensors(self, kind='topomap', show=True):
```

Comment:
I'd rather add picks parameter. This function doesn't really use the positions in layout, but the actual positions in the info structure. It would feel silly to have a mandatory info param and then ignore it when layout is provided. The user can still do ``mne.channels.read_layout('Vectorview-grad').plot()``.